VictoriaAlbert/TableViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  TableViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Marty Avedon on 11/10/16.
+//  Copyright © 2016 Marty Hernandez Avedon. All rights reserved.
+//
+
+import UIKit
+
+let endpoint = "http://www.vam.ac.uk/api/json/museumobject/search?q=skull"
+
+class TableViewController: UITableViewController {
+    
+    var selectedBric: BricABrac?
+    var bricArr:[BricABrac]? = [BricABrac]()
```

Comment:
This line has a couple competing ideas. You're declaring ```bricArr``` to be an Optional array of ```BricABrac``` but then you initialize it to an emty array of ```BricABrac```. That's not the same as nil. So, unless you plan to nil it out later (which I don't think you do or have to) you can remove the optional aspect of it. Then it's a little easier to deal with and look at later in the code.